### PR TITLE
Fix title column width on switch from horizontal to vertical layout 

### PR DESCRIPTION
**Pull request by barijaona**
Fix issue #1016

### CODE REVIEW

File: src/ArticleListView.m [hunk 1]
```diff
@@ -857,8 +856,10 @@ -(void)setOrientation:(NSInteger)newLayout
 {
 	isChangingOrientation = YES;
 	tableLayout = newLayout;
+	splitView2.autosaveName = nil;
```
Comment (josh64x2) [inline src/ArticleListView.m]: Can you please remove the autosaveName from Interface Builder too? That way a future contributor won't look at IB and get confused as to why the autosaveName isn't being used 😄 

screen shot 2017-09-20 at 10 40 23

Comment (barijaona) [reply src/ArticleListView.m]: Done.

**Comment by josh64x2** [comment]
Nice solution 😄

**Comment by barijaona** [comment]
The solution seems more complicated than what is needed, as there are redundant informations between the `Vienna3ReportLayoutColumns`/`Vienna3CondensedLayoutColumns`autosaves and the entry  in the preferences file with `MAPref_ArticleListColumns` ("MessageColumns") key.

But it works, so let's live with it.

**Pull request merged**